CK2Plus_expanded/events/000_CK2Plus_startup_events.txt.txt

```diff
@@ -0,0 +1,62 @@
+
+namespace = PlusStartup
+
+#######################################
+#
+# CK2Plus Startup Events
+#
+#######################################
+#
+# CATALOG
+#
+# 001 - 100: General events
+# 101 - 200: Toby Determined
```

Comment:
Who is Toby 😂

Reply:
Oops my bad!
Thought I'd have removed that... it's a sort of pun.
Have been watching too much of that show lately... it's sneaking in everywhere! But Disney is trying to steal it from Netflix for their stupid Disney Plus, so who can blame me?

Reply:
Feel free to remove that.
Would it be removed from the pull request if I fix it locally and then push the branch?

Reply:
Yep!

Reply:
Okay, it's gone.